packages/server/test/integration/utils/index.ts

```diff
@@ -8,14 +8,33 @@ export let app: App
 export const setupApp = async () => {
   process.env.SKIP_LOAD_ENV = 'true'
   process.env.SUPPRESS_LOGGING = 'true'
-  process.env.DATABASE_URL = path.join(__dirname, '../../../../../test/.test-data', `${uuidv4()}.sqlite`)
+  process.env.DATABASE_URL =
+    process.env.DATABASE_URL || path.join(__dirname, '../../../../../test/.test-data', `${uuidv4()}.sqlite`)
+
+  if (process.env.DATABASE_URL.startsWith('postgres')) {
+    process.env.DATABASE_SUFFIX = `__${randomLetters(8)}`
+    process.env.DATABASE_TRANSIENT = 'true'
+  }
 
   app = new App()
   await app.setup()
   await app.postSetup()
   return app
 }
 
+export const destroyApp = async () => {
+  await app.destroy()
+}
```

Comment:
This does not seem to be used anywhere!

Reply:
Good catch I did this before making DATABASE_TRANSIENT